pm.mirko.Atoms.yml

```diff
@@ -0,0 +1,199 @@
+  sources:
+    - type: git
+      url: https://github.com/AtomsDevs/atoms-core.git
+      commit: 7fbd7c193bbc0bcb76458d1a5370e1bc20e80e07
+  modules:
+  - name: orjson
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps orjson-*.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/d6/79/174c198de7beb2f5ce06c41ecafb1aafc0714bb61a051ad9dfbc2fed76e8/orjson-3.7.8-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+        sha256: 7ef66e7c47e9531dfa3b3ffb791b548c84903570b0914b01acc5eeb56ff5bc33
+  - name: python-requests
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://github.com/psf/requests/archive/refs/tags/v2.28.0.tar.gz
+        sha256: 6b757ca284d8dc192794010c960b0cf405fcc7aed9ed2e04a8269dc41b1231ad
+  - name: python-urllib3
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://github.com/urllib3/urllib3/archive/refs/tags/1.26.9.tar.gz
+        sha256: 5860049db8811e7a26e5f228359522814dc8159e6ec4f352b1b596e0dee80503
+  - name: python-chardet
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://github.com/chardet/chardet/archive/4.0.0.tar.gz
+        sha256: 67bbe3eb02691d38831ddf2719c40fe09381bc17f15321946f6cd71dc274e530
+  - name: python-certifi
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://github.com/certifi/python-certifi/archive/refs/tags/2021.10.08.tar.gz
+        sha256: 51c2cf76f7f2582d9c4eb5440bb311ef523ba0b319d0a973450ecebfb6802a3a
+  - name: python-idna
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    sources:
+      - type: archive
+        url: https://github.com/kjd/idna/archive/refs/tags/v3.3.tar.gz
+        sha256: 34b145ffe35b7872e9a3e0abc3a3a8330db90a0fd58db978725fd05c5c7d9f88
+
+- name: atoms-cli
+  buildsystem: simple
+  build-commands:
+    - python3 setup.py install --prefix=/app --root=/
+  sources:
+    - type: git
+      url: https://github.com/AtomsDevs/atoms-cli.git
+      commit: f0cf3718b517074ff63e1e7b54bf5d09d4b6e826
+  modules:
+    - name: python-tabulate
+      buildsystem: simple
+      build-commands:
+        - python3 setup.py install --prefix=/app --root=/
+      sources:
+        - type: archive
+          url: https://github.com/astanin/python-tabulate/archive/refs/tags/v0.8.10.tar.gz
+          sha256: 7a81a625d3cdfdd7c9db25f992410f46e7cbb20ed151126fceeac95586b90284
+
+- name: atoms
+  builddir: true
+  buildsystem: meson
+  build-commands:
+    - mkdir -p /app/lib/i386-linux-gnu
```

Comment:
```suggestion
```

Reply:
For some reason removing this line I am no longer able to build the flatpak from my pc but I think it is not a problem if the build is made by flathub